samples/sensor/accel_polling/sample.yaml

```diff
@@ -25,3 +25,18 @@ tests:
       - b_l4s5i_iot01a                  # lsm6dsl
       - sensortile_box                  # lis2dw12, lsm6dso, iisdhhc
       - thingy53/nrf5340/cpuapp         # adxl362, bmi270
+
+  sample.sensor.accel_polling.nrf54h:
+    depends_on: spi
+    tags: drivers spi sensors
+    filter: dt_alias_exists("accel0")
+    harness: console
+    harness_config:
+      fixture: PCA63566
```

Comment:
Not sure if sensor board marking (PCA63565 / PCA63566) shall be used or just 'sensor_adxl362'.

Reply:
I've decided to switch fixture name to sensor_adxl362. In that case same test can be used for nrf54h and nrf54l targets while different PCA6356x have to be fitted on top of Development Kits.

Reply:
obsolete